core/spec/models/spree/promotion_handler/coupon_spec.rb

```diff
@@ -90,6 +90,24 @@ module PromotionHandler
                 expect(order.reload.total).to eq(100)
               end
 
+              context 'and first line item is not promotionable' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.